configs/coins/arbitrum_archive.json

```diff
@@ -0,0 +1,67 @@
+{
+  "coin": {
+    "name": "Arbitrum Archive",
+    "shortcut": "ETH",
+    "label": "Arbitrum",
+    "alias": "arbitrum_archive"
+  },
+  "ports": {
+    "backend_rpc": 8306,
+    "backend_p2p": 38406,
+    "blockbook_internal": 9206,
+    "blockbook_public": 9306
+  },
+  "ipc": {
+    "rpc_url_template": "ws://127.0.0.1:{{.Ports.BackendRPC}}",
+    "rpc_timeout": 25
+  },
+  "backend": {
+    "package_name": "backend-arbitrum-archive",
+    "package_revision": "satoshilabs-1",
+    "system_user": "arbitrum",
+    "version": "3.0.3",
+    "docker_image": "offchainlabs/nitro-node:v3.0.3-3ecd01e",
+    "verification_type": "docker",
+    "verification_source": "6bfb702e99d66db88658e712ed0d8691abe03797afe436eaab60a6f31e627e0d",
+    "extract_command": "docker cp extract:/home/user/target backend/target; docker cp extract:/home/user/nitro-legacy backend/nitro-legacy; docker cp extract:/usr/local/bin/nitro backend/nitro",
+    "exclude_files": [],
+    "exec_command_template": "/bin/sh -c '{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/arbitrum_exec.sh 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
```

Comment:
`arbitrum_exec.sh` file does not exist, it should be `arbitrum_archive_exec.sh`

Reply:
https://github.com/trezor/blockbook/pull/1112/commits/85ca2abefa67baaafa1b4125e07d509edbf68533

Reply:
https://github.com/trezor/blockbook/pull/1112/commits/85ca2abefa67baaafa1b4125e07d509edbf68533